examples/frames/README.md

```diff
@@ -1,27 +1,23 @@
 # frames
 
-FIXME: description
-
-## Installation
-
-Download from http://example.com/FIXME.
-
 ## Usage
 
-FIXME: explanation
+### Start dockerized PostgreSQL
+
+    docker-start.sh
```

Comment:
this docker-start.sh can be a dev-env.sh

Reply:
> this docker-start.sh can be a dev-env.sh ?

Yeah, sure. I go further, it can be postgres-start.sh, because in plans we will have redis too.

Reply:
Agree, I imagine quite soon we may have something like cache (Redis) or other services running along with the web server.